### PR TITLE
Daily workflow to build examples

### PR DESCRIPTION
**Pull request by hellwolf**

### CODE REVIEW

File: .github/workflows/build-examples.yml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install, build and test
+        working-directory: examples/${{ matrix.example }}
+        run: |
+          yarn install --frozen-lockfile
+          yarn build
+          yarn test
+
+      # - name: Deployment
```
Comment (pi0neerpat) [inline .github/workflows/build-examples.yml:43]: can remove?

File: .github/workflows/build-examples.yml [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      - name: (force-install) Install, build and test
+        working-directory: examples/${{ matrix.example }}
+        run: |
+          yarn install --force
+          yarn build
+          yarn test
+          if [ -f "scripts/deploy.js" ];then
+            RELEASE_VERSION=master npx truffle --network goerli exec scripts/deploy.js
+          fi
+        env:
```
Comment (pi0neerpat) [inline .github/workflows/build-examples.yml:72]: every example may not have deploy script

File: .github/workflows/build-examples.yml [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+      # - name: (force-install) Deployment
+      #   working-directory: examples/${{ matrix.example }}
+      #   run: |
+      #     if [ -f "scripts/deploy.js" ];then
+      #       npx truffle --network goerli exec scripts/deploy.js
+      #     fi
+      #   env:
+      #     RELEASE_VERSION: master
+      #     GOERLI_MNEMONIC: ${{ secrets.BUILD_AGENT_MNEMONIC  }}
+      #     GOERLI_PROVIDER_URL: ${{ secrets.GOERLI_PROVIDER_URL }}
```
Comment (pi0neerpat) [inline .github/workflows/build-examples.yml:94]: cleanup?

**Review by pi0neerpat — APPROVED** [review]
LGTM just some minor non-blocking comments

File: .github/workflows/build-examples.yml (hunk 3 above)
Comment (hellwolf) [reply .github/workflows/build-examples.yml:94]: will leave it there until I find out what happens..

File: .github/workflows/build-examples.yml (hunk 2 above)
Comment (hellwolf) [reply .github/workflows/build-examples.yml]: removed

**Pull request merged**